log4j-api/src/main/java/org/apache/logging/log4j/Logger.java

```diff
@@ -137,7 +137,7 @@
      * Logs a message CharSequence with the {@link Level#DEBUG DEBUG} level.
      *
      * @param marker the marker data specific to this log statement
-     * @param message the message CharSequence to log.
+     * @param message the message CharSequence to log. THIS MAY OR MAY NOT BE INTERPRETED AS A MESSAGE OBJECT
```

Comment:
-1 on the UPPER CASE SHOUTING. In addition, the text is not really helpful, it does not tell you why this might matter.